### ISSUE TITLE
Nerdfont Icons partially cut on workspaces (i3)

### ISSUE THREAD

**Issue by h1ghrise** [polybar/polybar#2045]
## active workspaces affected
all of a sudden, the Icon displaying the **active** workspace, and ones which have notifcations on it are displayed incorrectly.
Inactive workspaces are displayed correctly.

Screenshot:
![Imgur](https://i.imgur.com/uvNzQxW.png)

**Expected behavior:**

display all icons in the same way.

**Actual behavior:**

active workspace icon is partially cut.

*What actually happens*

**Was it working before?**
yes, changed all of a sudden. Suspect a Nerdfont update.
Version                   : 2.1.0-1 (installed March 20)
Installed on:         : Fr 20 Mar 2020 21:33:35

## To Reproduce
*A minimal but complete config with which the problem occurs:*
```dosini
; MWE for Issue creation
;

[colors]
background = #ffffff

foreground = ${xrdb:foreground} 
foreground-alt = ${xrdb:color7} 

; Mappings from Xresources

color0 = ${xrdb:color0}   
color1 = ${xrdb:color1}   
color2 = ${xrdb:color2}   
color3 = ${xrdb:color3}   
color4 = ${xrdb:color4}   
color5 = ${xrdb:color5}   
color6 = ${xrdb:color6}   
color7 = ${xrdb:color7}   
color8 = ${xrdb:color8}   
color9 = ${xrdb:color9}   
color10 = ${xrdb:color10} 
color11 = ${xrdb:color11} 
color12 = ${xrdb:color12} 
color13 = ${xrdb:color13} 
color14 = ${xrdb:color14} 
color15 = ${xrdb:color15} 

primary = ${xrdb:color3:#fff} 
secondary = ${xrdb:color4:#e60053}
alert = ${xrdb:color5:#bd2c40}
warning = ${xrdb:color1:#ff8800}

[bar/mwe]
width = 95%
height = 25
offset-x = 2%
radius = 2
module-margin = 2


font-1 = SauceCodePro Nerd Font:pixelsize=13;1
font-0 = "Google Sans Medium:size=10;3"
modules-left = i3
tray-position = right
tray-padding = 2
override-redirect = true

cursor-click = pointer
cursor-scroll = ns-resize

[module/i3]
type = internal/i3
format = <label-state> <label-mode>
index-sort = true
wrapping-scroll = false

ws-icon-0 = 1;
ws-icon-1 = 2;
ws-icon-2 = 3;
ws-icon-3 = 4;ﭮ
ws-icon-4 = 5;

label-mode-padding = 2
label-mode-foreground = ${colors.color6}
label-mode-background = ${colors.color3}

; focused = Active workspace on focused monitor
label-focused = %icon%
label-focused-background = ${colors.color1}
label-focused-foreground = ${colors.color7}
label-focused-underline = ${colors.background}
label-focused-padding = 2

; unfocused = Inactive workspace on any monitor
label-unfocused = %icon%
label-unfocused-padding = 2
; visible = Active workspace on unfocused monitor
label-visible = %icon%
label-visible-background = ${self.label-focused-background}
label-visible-underline = ${self.label-focused-underline}
label-visible-padding = ${self.label-focused-padding}

; urgent = Workspace with urgency hint set
label-urgent = %icon%
label-urgent-background = ${colors.color7}
label-urgent-padding = 2
```
## Polybar Log
launched MWE with `polybar mwe -c ~/.config/polybar/mwe --log=trace`

```
* Loading config: /home/h1ghrise/.config/polybar/mwe
- config: Current bar section: [bar/mwe]
* Loaded monitor eDP1 (2160x1440+0+0)
* Bar geometry: 2052x25+43+0; Borders: 0,0,0,0
- bar: Attach X event sink
- bar: Attach signal receiver
- controller: Install signal handler
- controller: Setup user-defined modules
* Found matching X resource "color1" (value=#C65A56)
* Found matching X resource "color7" (value=#cfc7d1)
* Found matching X resource "color1" (value=#C65A56)
* Found matching X resource "color7" (value=#cfc7d1)
* Found matching X resource "color3" (value=#395987)
* Found matching X resource "color6" (value=#7289B3)
* Starting application
- controller: Main thread id = 1
* Starting module/i3
* Entering event loop (thread-id=1)
- module/i3: Thread id = 2
* Eventqueue worker (thread-id=3)
- bar: Create renderer
- renderer: Get TrueColor visual
- renderer: Allocate colormap
- renderer: Allocate output window
- renderer: Allocate window pixmaps
- renderer: Allocate graphic contexts
- renderer: Allocate alignment blocks
- renderer: Allocate cairo components
- renderer: Load fonts
* Configured DPI = 96x96
* Loaded font "Google Sans Medium:size=10" (name=Bitstream Vera Sans, offset=3, file=/usr/share/fonts/TTF/Vera.ttf)
* Loaded font "SauceCodePro Nerd Font:pixelsize=13" (name=SauceCodePro Nerd Font, offset=1, file=/usr/share/fonts/TTF/Sauce Code Pro Nerd Font Complete.ttf)
* Bar window: 0x0800001
- bar: Reconfigure window
- bar: Set window WM_NAME
- bar: Set window _NET_WM_WINDOW_TYPE
- bar: Set window _NET_WM_STATE
- bar: Set window _NET_WM_DESKTOP
- bar: Set window _NET_WM_PID
- bar: Map window
- bar: Received expose event
- bar: Draw empty bar
- bar: Setup tray manager
- tray: Find systray selection atom for the default screen
* Activating tray manager
- tray: Create tray window
* Tray window: 0x0800008
- tray: Restacking tray window
- bar: Set window WM_NAME / WM_CLASS
- tray: Set window WM_PROTOCOLS
- tray: Set window _NET_WM_WINDOW_TYPE
- tray: Set window _NET_WM_STATE
- tray: Set window _NET_WM_PID
- tray: Set window _NET_SYSTEM_TRAY_VISUAL
- tray: Set window _NET_SYSTEM_TRAY_ORIENTATION
- tray: Set _NET_SYSTEM_TRAY_COLORS to ff000000
- tray: Change selection owner to 0x0800008
- tray: visibility_change (state=1, activated=1, mapped=0, hidden=0)
* module/i3: Rebuilding cache
- bar: Force update
* Redrawing bar window
- renderer: flush(1 geom=78x25+0+0, falloff=0)
* Notifying pending tray clients
- tray: Received client_message
* Processing docking request from 0x140006b
- tray: Get client _XEMBED_INFO
- tray: Update client window
- tray: Configure client size
- tray: Add client window to the save set
- tray: Reparent client
- tray: Send embbeded notification to client
- tray: Map client
- tray: Received client_message
* Processing docking request from 0x0c00145

```

## Screenshots
*If applicable, add screenshots to help explain your problem.*

## Environment:
* WM: i3
* Distro:  OS: Manjaro Linux x86_64, Kernel 5.5.11-1-MANJARO
* Output of `polybar -vvv`:
```
polybar 3.4.2

Features: +alsa +curl +i3 +mpd +network(libnl) +pulseaudio +xkeyboard

X extensions: +randr (+monitors) +composite +xkb +xrm +xcursor

Build type: Release
Compiler: /usr/bin/c++
Compiler flags: -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -fno-plt -Wall -Wextra -Wpedantic -O3 -DNDEBUG
Linker flags: -Wl,-O1,--sort-common,--as-needed,-z,relro,-z,now 
```
## Additional context
Fonts are used system-wide, and have no display issues somewhere else.
Used font:
`* Loaded font "SauceCodePro Nerd Font:pixelsize=13" (name=SauceCodePro Nerd Font, offset=1, file=/usr/share/fonts/TTF/Sauce Code Pro Nerd Font Complete.ttf)
`
tried also with `Hack Nerd Font`- same outcome.

**Comment by h1ghrise** [comment]
might be a duplicate of #991

**Comment by h1ghrise** [comment]
resolved by reverting back to Nerdfont 2.0

**Comment by adho08** [comment]
You don't really find any discrete answer in the links you provided. So I'll tell you what worked for me. 
I specified the name/icon of the workspace inside my i3 config. So inside my polybar config: `label-focused = %name%` also for unfocused, visible and urgent workspaces. But for me, I just had to not specify a background color and the glyph was fully visible. Unfortunately, this approach isn't really elegant and I don't think this behaviour is intended by the polybar devs.
